io.github.daid.EmptyEpsilon.metainfo.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 daid -->
+<component type="desktop-application">
+  <id>io.github.daid.EmptyEpsilon</id>
+  
+  <name>EmptyEpsilon</name>
+  <summary>Spaceship bridge simulator game</summary>
+  
+  <!-- of course I just copied the stuff from the website, daid needs to agree first <metadata_license>FSFAP</metadata_license>-->
+  <project_license>GPL-2.0-only</project_license>
```

Comment:
This isn't true. Some of the assets are under a closed license.

Reply:
Ok, now having looked at all files with license in their name, I suppose the correct one would be:
`GPL-2.0-only AND LicenseRef-TurboSquid-Royalty-Free-License AND OFL-1.1 AND Apache-2.0`